src/module-elasticsuite-tracker/view/frontend/templates/config.phtml

```diff
@@ -14,34 +14,56 @@
  * @license   Open Software License ("OSL") v. 3.0
  */
 ?>
-<?php /** @var $block Smile\ElasticsuiteTracker\Block\Config **/ ?>
-<?php $jsonHelper = $block->getJsonHelper(); ?>
-<?php if ($block->isEnabled()) : ?>
-<script>
+<?php
+/**
+ * @var $block Smile\ElasticsuiteTracker\Block\Config
+ * @var $secureRenderer Magento\Framework\View\Helper\SecureHtmlRenderer
+ */
+?>
+<?php
+$jsonHelper = $block->getJsonHelper();
+if ($block->isEnabled()) {
+$beaconUrl = $this->escapeJsQuote($block->getBeaconUrl());
+$telemetryUrl = $this->escapeJsQuote($block->getTelemetryUrl());
+$telemetryEnabled = $this->escapeJsQuote($block->isTelemetryEnabled());
+$sessionConfig = $jsonHelper->jsonEncode($block->getCookieConfig());
+
+$scriptString = "
 //<![CDATA[
 try {
-    let trackerConfig = {
-        beaconUrl        : '<?= /* @noEscape */ $this->escapeJsQuote($block->getBeaconUrl()); ?>',
-        telemetryUrl     : '<?= /* @noEscape */ $this->escapeJsQuote($block->getTelemetryUrl()); ?>',
-        telemetryEnabled : '<?= /* @noEscape */ $this->escapeJsQuote($block->isTelemetryEnabled()); ?>',
-        sessionConfig    : <?= /* @noEscape */ $jsonHelper->jsonEncode($block->getCookieConfig()); ?>,
-    };
-<?php if ($block->isUsingAPI()): ?>
-    trackerConfig.endpointUrl = '<?= /* @noEscape */ $this->escapeJsQuote($block->getEndpointUrl()); ?>';
-<?php endif; ?>
-    smileTracker.setConfig(trackerConfig);
+    smileTracker.setConfig({
+        beaconUrl: '{$beaconUrl}',
+        telemetryUrl: '{$telemetryUrl}',
+        telemetryEnabled: '{$telemetryEnabled}',
+        sessionConfig: '{$sessionConfig}'
```

Comment:
@vahonc: I believe there is a bug introduced on this line, the single quotes should not be needed around `{$sessionConfig}`. If the json data contains a single quote, this will break. In the old code, there also weren't single quotes surrounding this value.

Reply:
My bad, it looks like @romainruaud already fixed this shortly after merging this, in https://github.com/Smile-SA/elasticsuite/commit/daff073c9e16f2e7a44a5b90cc9fc01d5f3fb4ce, thanks guys!

Reply:
Yes correct @hostep :)

Reply:
Sorry guys, it's just a typo, my bad